backend/handler/user.go

```diff
@@ -58,6 +58,15 @@ func (h *UserHandler) Create(c echo.Context) error {
 func (h *UserHandler) Get(c echo.Context) error {
 	userId := c.Param("id")
 
+	sessionToken, ok := c.Get("session").(jwt.Token)
```

Comment:
Why? It is "protected" by the session middleware, wha do we need this in addition to the middleware?